azerothcore/azerothcore-wotlk
# feat(Core/Map): Allow disabling the blizzlike PvP line of sight checks

Has to do with M2 object occlusion



## Changes Proposed:
-  Allow skipping the blizzlike los checks in pvp, so players can no longer cast spells through trees and other objects
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


Log two characters, toggle debug bg,  queue wsg, try to cast a spell through the treestumps
Go to BWL, check it supression traps still work (not tested)

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
